corn_app/preprocess.py

```diff
@@ -0,0 +1,93 @@
+"""
+preprocess.py
+Resizes images to a standard size
+"""
+from PIL import Image
+import sys
+import os
+
+STANDARD_WIDTH = 1280 # This is our standard image width
+STANDARD_HEIGHT = 720 # This is our standard image height
+
+def standardize(image):
+    """
+    Standardizes image parameters.
+    For size parameters, we are standardizing to a resolution of 1280 x 720
+
+    Args:
+        image (Image): An open Image instance.
+
+    Returns:
+        bool: True if image was resized, false otherwise.
+    """
+    if not image:
+        print('No Image Found')
+        return False
+
+    print('Resizing Image...')
+
+    image = image.resize((STANDARD_WIDTH, STANDARD_HEIGHT))
+    return image
```

Comment:
There is no need to return ```image``` here.  The ```image``` variable is a pointer to the object on the python heap. The ```image``` argument will be changed after the function call if anything is done to it. Just return ```True``` or ```False``` in this function.

Reply:
@shawnhugginsjr Turns out this isn't true. I printed out the reassigned image object within the function and the image object outside the function and they point to different places in memory. I had it returning true/false on the last commit. When you guys left class today, I realized that the image wasn't being resized.

Reply:
You're right, I just checked the documentation. The resize isn't done in place. In that case, assigning image the resized image is fine.